src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Shape {
```

Comment:
```suggestion
public enum FegureName {
```

Reply:
Why Shape is a bad name?

Reply:
Because you named elements the same way as you named classes. But they are figures and this is Shapes, why not vice versa? Or why don't you use the same word? I guess that is because it will look weird if an enum and an abstract class have the same name. 
Also, we know that the FegureName is a property of Figure, so the name i suggested is perfect for us.